governance.md

```diff
@@ -0,0 +1,200 @@
+# Governance
+
+This document describes the rules and governance of the project. It is a slightly modified version of the [Prometheus Governance](https://prometheus.io/governance/#governance-changes).
+
+It is meant to be followed by all the developers of the Prometheus Operator project and the Prometheus Operator community. Common terminology used in this governance document are listed below:
+
+* **Maintainers Team**: A core Prometheus Operator team that have owner access to http://github.com/prometheus-operator organization and all projects within it. Current list is available [here][maintainers-doc].
+
+* **Triage Team**: Contributors who does not belong to Maintainer's team, but has `Triage` GitHub role on [Prometheus Operator](https://github.com/prometheus-operator) repository allowing to change GitHub issues and PRs statuses and labels.
+They are listed [here](https://github.com/prometheus-operator/prometheus-operator/blob/master/MAINTAINERS.md#triage).
+
+* **The Prometheus Operator project**: The sum of all activities performed under the [prometheus-operator organization on GitHub][gh], concerning one or more repositories or the community.
+
+Both Triage and Maintainers are part of [`prometheus-operator-team@googlegroups.com`][team] email list.
+
+## Values
+
+The Prometheus Operator developers and community are expected to follow the values defined in the [Code of Conduct][coc].
+
+Furthermore, the Prometheus Operator community strives for kindness, giving feedback effectively, and building a welcoming environment. The Prometheus Operator developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Decision making
+
+### Maintainers Team
+
+Team member status may be given to those who have made ongoing contributions to the Prometheus Operator project for at least 3 months.
+This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account.
```

Comment:
I would like to say, besides code improvements to me the person also needs to review PRs and issues of the code they added. Otherwise, to me, that is not a maintainer but a reviewer. Any objections to that?

Reply:
done, added `..., pull-request reviews, issue triaging, ...`